subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/util/PomCustomizerTest.groovy

```diff
@@ -186,6 +186,41 @@ class PomCustomizerTest extends Specification {
 """
     }
 
+    def "CI management from configuration"() {
```

Comment:
please also cover a default value ciManagement url and system

Reply:
Added in next commit